requirements.yml

```
@@ -4,5 +4,7 @@ collections:
     version: 1.3.0
     type: galaxy
   - name: community.general
-    version: 5.3.0
+    version: 8.6.1
     type: galaxy
+  - name: community.docker
+    version: 3.10.1
```

```suggestion
    version: 3.11.0
```
Might as well install the latest